tools/serverpod_cli/lib/src/util/model_helper.dart

```diff
@@ -88,10 +88,14 @@ class ModelHelper {
   ) async {
     List<FileSystemEntity> modelSourceFileList = [];
 
-    var dir = Directory('$separator${joinAll(absolutePathParts)}');
+    var path = joinAll(absolutePathParts);
+
+    if (!Platform.isWindows) {
+      path = '$separator$path';
+    }
```

Comment:
Another fix here, but do you actually need the separator to define it as an absolute path?

Reply:
On mac yes I do! But it could very well be a problem on linux not sure.